### DISCUSSION TITLE
[Question][Hw4] About submission

### DISCUSSION THREAD

**Discussion by dlwjdwo00701** [prosyslab-classroom/cs524-program-analysis#303] [Q&A]
Name: jungjae Lee

Hello, I'm Jungjae Lee with the ID 20234335.

I barely managed to complete my assignment by submitting the coverage report at 00:15 on April 23, but I missed the submission deadline......

Although I uploaded the completed code to GitHub after the deadline(00:22), is there any possibility that it could still be graded?

I worked tirelessly for days, but it's disappointing.

I sent an email yesterday regarding this matter, but I'm reaching out again on the issue board.

Thank you.

**Comment by goodtaeeun** [comment]
I am very sorry but submissions past the late deadline cannot be graded.
However, completing the assignment will help you do your next assignment and thus will not be meaningless.
I hope this cheers you up.

**Comment by dlwjdwo00701** [comment]
Thank you for your answer!
